plugins/graphql-java-codegen-gradle-plugin/graphql-codegen-gradle-plugin/build.gradle

```diff
@@ -0,0 +1,48 @@
+plugins {
+    id "com.gradle.plugin-publish" version "0.11.0"
+    id "java-gradle-plugin"
+}
+
+repositories {
+    maven {
+        url 'https://repo.gradle.org/gradle/libs-releases-local'
+    }
+    mavenCentral()
+    mavenLocal()
+    jcenter()
+}
+
+apply plugin: "java"
+apply plugin: "idea"
+apply plugin: "maven-publish"
+
+group = "io.github.kobylynskyi"
+version = "1.5.0-SNAPSHOT"
+description = "Provides a task for generating Java code based on GraphQL schema"
+
+dependencies {
+    compile gradleApi()
+
+    compile 'io.github.kobylynskyi:graphql-java-codegen:1.5.0-SNAPSHOT'
```

Comment:
This should use the current version to work, right? Maybe it should be a variable then, because otherwise we'll need to change the version in too many places when a release is made.

Reply:
Thanks. That's a good point.
I've removed the hardcoded version wherever possible. In other places, I don't see how we can refer to an external file.